features/support/env.rb

```diff
@@ -1,60 +1,17 @@
-# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
-# It is recommended to regenerate this file in the future when you upgrade to a
-# newer version of cucumber-rails. Consider adding your own code to a new file
-# instead of editing this one. Cucumber will automatically load all features/**/*.rb
-# files.
-
+#edit at your own peril.
 require 'cucumber/rails'
 require 'cucumber/rspec/doubles'
 
-# Capybara defaults to CSS3 selectors rather than XPath.
-# If you'd prefer to use XPath, just uncomment this line and adjust any
-# selectors in your step definitions to use the XPath syntax.
-# Capybara.default_selector = :xpath
 
-# By default, any exception happening in your Rails application will bubble up
```

Comment:
I'm glad we're getting rid of all these unnecessary lines! Takes us one step closer to clean, readable code.